scripts/compute_mappings/README.md

```diff
@@ -4,4 +4,6 @@ The python scripts in this directory are generated via cli command:
 jupyter nbconvert <notebook_filename>.ipynb --to python
 ```
 
-To update scripts, edit the notebooks and re-run the cli command to re-generate the python scripts.
\ No newline at end of file
+To update scripts, edit the notebooks and re-run the cli command to re-generate the python scripts.
```

Comment:
Update: after discussion with Pablo / Clever Canary, I don't think there's a desire to keep the notebooks. So we can delete them and remove this guidance from the README